sdk/core/Azure.Core/src/RetryOptions.cs

```diff
@@ -35,5 +35,10 @@ internal RetryOptions()
         /// The approach to use for calculating retry delays.
         /// </summary>
         public RetryMode Mode { get; set; } = RetryMode.Exponential;
+
+        /// <summary>
+        /// The timeout for a single operation.
+        /// </summary>
+        public TimeSpan TryTimeout { get; set; } = TimeSpan.FromSeconds(100);
```

Comment:
cc @KrzysztofCwalina 
I took the default HttpClient timeout.

Reply:
I find the name a little hard to wrap my head around at first glance.  `PerTryTimeout` is kind of awkward but matches the pipeline position naming with `PerCall`, etc.   I don't feel strongly here.

Reply:
Changed to `NetworkTimeout`